<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>791</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit_Input">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>701</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Output">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>701</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Mes">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>530</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>2000000</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Port">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Adress">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Our_Port">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>260</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Порт удаленный</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Adress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес собеседника</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Our_Port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Наш порт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Send_Mes">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>530</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Create">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Show_Adress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Show_Adress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>432</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать мой IP-адрес</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_you">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Вы:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>-5</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Собеседник:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clear_All">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>600</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить всё</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Disconnect">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отсоединиться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Spam">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>600</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начать спам</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_StopSpam">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>600</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Остановить спам</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
